src/fty_security_wallet.cc

```diff
@@ -33,16 +33,18 @@
 
 #include "fty_common_socket_basic_mailbox_server.h"
 
+#include <systemd/sd-daemon.h>
```

Comment:
Should this include be protected by an #if clause

Reply:
Hm, misread git-grep, thanks.

Reply:
Actually we should not need it here, it is in the chain of common include files of the component (which honor the optionality).